components/inputs/input-time.js

```diff
@@ -47,6 +47,11 @@ export function getDefaultTime(time, enforceTimeIntervals, timeInterval) {
 			return END_OF_DAY;
 		case 'startOfDay':
 			return enforceTimeIntervals ? MIDNIGHT : START_OF_DAY;
+		case 'now': {
+			const today = getToday();
```

Comment:
Added a 'now' case for getDefaultTime that will return the current time without rounding to the nearest interval.

Reply:
Technically I think this means they can go `<d2l-input-time value="now">`, right? Not sure if we actually want that since that's the same as not setting a value at all I believe.

Also, how does `enforce-time-invervals` work with this?

Reply:
This affects the default-value attribute in the time and date-time picker, not the value. So consumers could set default-value="now', but that functionality is currently not exposed.

As for `enforce-time-intervals`. the date-time input will always have that property set to false, so we technically couldn't get to a case where we needed to enforce them here. The question is if we do want to expose the now option to consumers or keep it hidden.

Reply:
Ah right, `default-value`.

Makes sense about `enforce-time-intervals`. I'm good keeping it hidden until a valid use case comes up.

Reply:
I ended up just adding a ternary statement to enforce the intervals in the case the now option becomes exposed in the future. We can assume it won't be used for now and always return the date since it's only used in date-time.